dataverse-webapp/src/main/resources/config/dataverse.default.properties

```diff
@@ -126,9 +126,15 @@ AntivirusScannerSocketPort=3310
 AntivirusScannerSocketTimeout=60000
 
 FilesIntegrityCheckTimerExpression=0 0 1 * *
+
 # Following three options depend on application used.
 # E.g for unrar these will be: unrar/l/-
 RarDataUtilCommand=
 RarDataUtilOpts=
 RarDataLineBeforeResultDelimiter=
+
+# Max sizes of input and output file for calculation of gzip uncompressed size
+GzipMaxInputFileSizeInBytes=
+GzipMaxOutputFileSizeInBytes=
```

Comment:
I believe we usually have default values for settings - zeroes here?. This way we skip null checks in code. 
Btw. don't we want some `non-zero` default values already?

Reply:
In that case I prefer to be over-defensive: any empty value in properties will result in errors during upload of gzipped files.

Reply:
However I've added zeros as default values.